src/GraphQLParser.Tests/LexerTests.cs

```diff
@@ -31,7 +31,7 @@ public void Lex_ATPunctuation_HasCorrectStart()
         public void Lex_ATPunctuation_HasCorrectValue()
         {
             var token = GetATPunctuationTokenLexer();
-            token.Value.ShouldBeNull();
+            token.Value.ToString().ShouldBe("@");
```

Comment:
It is worth paying attention to this. I began to return the actual contents of the token, not null.